marketplace-service/src/main/java/com/axonivy/market/constants/PreviewConstants.java

```diff
@@ -0,0 +1,13 @@
+package com.axonivy.market.constants;
+
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public class PreviewConstants {
+
+  public static final String PREVIEW_DIR = "marketplace-service/release-preview";
```

Comment:
@quanpham-axonivy if you set marketplace-service/data/work, it will create another folder marketplace-service instead of using current data folder. Please check FileDownloadService.ROOT_STORAGE_FOR_PRODUCT_CONTENT
![image](https://github.com/user-attachments/assets/a1ab37f8-9e9c-4685-85e0-b46af3702180)